tools/prompt_generator/prompt.py

```diff
@@ -56,6 +111,9 @@ def format_prompt_template_listener(system_prompt, user_prompt_1, assistant_resp
                         hf_output = gr.Textbox(label="Using HF Transformers", interactive=False, min_width=600, lines=25)
 
     system_prompt.change(format_prompt_template_listener, [system_prompt, user_prompt_1, assistant_response, user_prompt_2], [prompt_output, python_output, hf_output])
```

Comment:
Convert second and third parameter to a variable.

Reply:
Do the same thing for the remaining lines.

Reply:
I'm not sure I follow what do you mean with converting the second and thid param to a variable? Is is to put in the explicit `input` and `output` variable names in the call to the listener?